packages/katello/katello-repos/katello-repos.spec

```diff
@@ -1,6 +1,15 @@
 %global pulp_release stable
 %global pulp_version 2.15
 
+%if 0%{?suse_version}
+%define dist suse%{?suse_version}
+%define repo_dir %{_sysconfdir}/zypp/repos.d
+%define repo_dist sles12
```

Comment:
The distro for SLES12sp3 is 1315.  The numbers are in a chart, I'm not sure how we should handle this, since we used `sles12` as the directory for the repo URL.   Should we use suse1315? But then you'd need separate directories for each release of SLES.  :man_shrugging:

Reply:
i'm okay with this, but we need to add support for sles11 as well (waiting on https://github.com/theforeman/foreman-packaging/pull/2406 to publish the repo)

I don't think i want to get in the business for publishing for each sles minor version.

Reply:
Ok I updated it, so everything < 1300 is SLES11 otherwise SLES12.  We'll have to update this every time we support a new SLES version.